src/redux/books/booksSlice.js

```diff
@@ -1,18 +1,31 @@
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
-
-import { fetchBooks, addBook, removeBook } from '../../AddApi/Api';
-
-const initialState = {
-  books: [],
-  status: 'idle',
-  error: null,
-};
-
+import { fetchBooks, addBook, removeBook } from '../../BooksApi/BooksApi';
+
+const initialState = [
+  {
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  },
+];
 export const fetchBooksAsync = createAsyncThunk('books/fetchBooks', async () => {
   const response = await fetchBooks();
+  // console.log(response, 'this is api data');
```

Comment:
* Kindly remove all commented code form your code base as it makes your code bulky